docs/basics/prompting.md

```diff
@@ -3,12 +3,12 @@ sidebar_position: 1
 ---
 # 🟢 Prompting
```

Comment:
Careful, you are editing the English file :)

Reply:
Yes, sorry about that, already fixed

Reply:
Can you remove the edits on this file from the PR?